### PR TITLE
Release v1.21.5 build #1

### PR DESCRIPTION
**Pull request by adipietro18**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ source:
     - patches/0006-popcnt_fix.patch                     # [blas_impl == "mkl" and win]
 
 build:
-  number: 0
+  number: 1
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:19]: We haven't version 1.21.5 yet so the build number should be `0`

**Comment by skupr-anaconda** [comment]
We should update home url with HTTPS:
 `home: https://numpy.scipy.org/`
and fix the license name 
 `license: BSD-3-Clause`

File: recipe/meta.yaml (hunk 1 above)
Comment (adipietro18) [reply recipe/meta.yaml:19]: I believe we already have v1.21.5 on **win-64/32** & **linux-aarch64**:

```
conda search numpy=1.21.5 --subdir linux-aarch64
Loading channels: done
# Name                       Version           Build  Channel             
numpy                         1.21.5 py310h983bb6b_0  pkgs/main           
numpy                         1.21.5  py37hd490b01_0  pkgs/main           
numpy                         1.21.5  py38hd490b01_0  pkgs/main           
numpy                         1.21.5  py39hd490b01_0  pkgs/main 
```

Comment (skupr-anaconda) [reply recipe/meta.yaml:19]: Yeah, thanks! And atrifacts on win32 are built twice with build number 0:
https://repo.anaconda.com/pkgs/main/win-32
numpy-1.21.5-py310h54a06cd_0.tar.bz2
numpy-1.21.5-py310h75fcd9c_0.tar.bz2
numpy-1.21.5-py37h73ede0f_0.tar.bz2
numpy-1.21.5-py37hf27c404_0.tar.bz2
numpy-1.21.5-py38h73ede0f_0.tar.bz2
numpy-1.21.5-py38hf27c404_0.tar.bz2
numpy-1.21.5-py39h73ede0f_0.tar.bz2
numpy-1.21.5-py39hf27c404_0.tar.bz2

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — COMMENTED** [review]
The package looks good the only detail is that we need to use the `https` instead of `http` in the home url
https://github.com/AnacondaRecipes/numpy-feedstock/blob/v1.21.5-1/recipe/meta.yaml#L136

**Review by varlackc — COMMENTED** [review]
LGTM

**Review by varlackc — APPROVED** [review]

**Pull request merged**